File: ThaiTuanERP2025.Client/src/app/pages/finance/models/cash-out-group.dto.ts
export interface CashOutGroupDto {
      id: string;
      code: string;
      name: string;
      isActive: boolean;
      description?: string;
}
export type CreateCashOutGroupRequest = Omit<CashOutGroupDto, 'id' | 'isActive'> & { isActive?: boolean };
export type UpdateCashOutGroupRequest = Partial<Omit<CashOutGroupDto, 'id'>>;

File: ThaiTuanERP2025.Server/ThaiTuanERP2025.Application/Finance/DTOs/CashOutGroupDto.cs
namespace ThaiTuanERP2025.Application.Finance.DTOs
{
	public record CashoutGroupDto(
		Guid Id, string Code, string Name, string? Description, bool IsActive,
		Guid? ParentId, string? Path, int Level,
		List<CashoutGroupDto>? Children
	);
}
